dask/array/tests/test_fft.py

```diff
@@ -137,6 +137,7 @@ def test_wrap_ffts(modname, funcname, dtype):
 
     wfunc = fft_wrap(func)
     assert wfunc(darrc).dtype == func(nparrc).dtype
+    assert wfunc(darrc).shape == func(nparrc).shape
```

Comment:
Ok, well I just noticed that this caused a failure as I'm seeing shape mismatches with `*rfft` on SciPy. This actually doesn't surprise me as SciPy's functions preserve the original array's shape (up to `n`). However, NumPy's `*rfft`s don't.